IPs/RSKIP491.md

```diff
@@ -0,0 +1,59 @@
+---
+rskip: 491
+title: Reduce target difficulty to lower average block time to 10s
+description: This RSKIP proposes a 30% reduction in target difficulty to lower the average block time from 14 seconds to 10 seconds, improving transaction finality while maintaining network security.
+status: Draft
+purpose: Usa
+author: PDG (@patogallaiovlabs)
+layer: Core
+complexity: 1
+created: 2025-02-27
+---
+
+# Reduce target difficulty to lower average block time to 10s
+
+|RSKIP          | 491        |
+| :------------ |:-------------|
+|**Title**      |Reduce target difficulty to lower average block time to 10s |
+|**Created**    |27-FEB-2025 |
+|**Author**     |Patricio Gallardo|
+|**Purpose**    |Usa|
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes reducing the target difficulty by 30% to achieve an average block time of 10 seconds instead of the current 14 seconds. By adjusting the difficulty calculation, Rootstock can improve transaction finality while maintaining network security and stability.
+
+## Motivation  
+Currently, Rootstock maintains a 14-second block time, balancing security and efficiency.  
+
+Recent analysis of block template refresh intervals (as explored in [this research](https://blog.rootstock.io/noticia/leveraging-bitcoins-security-exploring-the-dynamics-of-merged-mining/)) showed that several mining pools have improved their block template refresh policies, reducing the time it takes to update new templates. These improvements have led to:  
+- a **decrease in the sibling-to-main block ratio**,  
+- and **faster effective block times**, previously averaging 30 seconds and now around [24 seconds](https://stats.rootstock.io/).  
```

Comment:
```suggestion
- and **faster effective block times**, previously averaging 30 seconds and now around [24 seconds](https://stats.rootstock.io/) (at the time of writing).  
```
Maybe add a comment that this was when you wrote the rskip so it survives the test of time (?)